WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -9491,10 +9491,11 @@
 			outputFileListPaths = (
 			);
 			outputPaths = (
+				"$(DERIVED_FILE_DIR)/check-for-nested-frameworks.txt",
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = "/bin/sh -euo pipefail";
-			shellScript = "${PROJECT_DIR}/../Scripts/check-nested-frameworks.sh\n";
+			shellScript = "${PROJECT_DIR}/../Scripts/check-nested-frameworks.sh\n\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
```

Comment:
What would happen on a clean build if the script fails? 🤔 Does Xcode exit as well (equivalent to `set -e`) or runs the `echo` command anyway?

https://github.com/woocommerce/woocommerce-ios/blob/021793264966a78e9e50b0e835a9d457ad385c3d/Scripts/check-nested-frameworks.sh#L24

Reply:
Oh that's a good point, Xcode will run the `echo` no matter if `check-nested-frameworks.sh`, so will always print "SUCCESS" to the output file.

Reply:
Oh, interesting.

In this case, we could write the result of the input file into the output file.

```
echo `${SCRIPT_INPUT_FILE_0}` > ${SCRIPT_OUTPUT_FILE_0}
```

Have you considered using [`tee`](https://man7.org/linux/man-pages/man1/tee.1.html)?

```
./${SCRIPT_INPUT_FILE_0} | tee ${SCRIPT_OUTPUT_FILE_0}
```

Also, I can't see an input file in the list in this diff. Perhaps this would achieve logging the script run result to file?

```
${PROJECT_DIR}/../Scripts/check-nested-frameworks.sh | tee ${SCRIPT_OUTPUT_FILE_0}
```

Reply:
> Have you considered using [tee](https://man7.org/linux/man-pages/man1/tee.1.html)?

I didn't know that one, thanks!